service/src/main/java/tds/config/repositories/impl/AccommodationsQueryRepositoryImpl.java

```diff
@@ -0,0 +1,187 @@
+package tds.config.repositories.impl;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import javax.sql.DataSource;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import tds.config.Accommodation;
+import tds.config.repositories.AccommodationsQueryRepository;
+
+@Repository
+public class AccommodationsQueryRepositoryImpl implements AccommodationsQueryRepository {
+    private final NamedParameterJdbcTemplate jdbcTemplate;
+
+    private static final String SEGMENTED_ASSESSMENT_SQL = "SELECT \n" +
+        "  distinct SegmentPosition as segment,\n" +
+        "  TType.DisableOnGuestSession as disableOnGuestSession, \n" +
+        "  TType.SortOrder as toolTypeSortOrder , \n" +
+        "  TT.SortOrder as toolValueSortOrder, \n" +
+        "  TType.TestMode as typeMode, \n" +
+        "  TT.TestMode as toolMode, \n" +
+        "  Type as accType, \n" +
+        "  Value as accValue, \n" +
+        "  Code as accCode, \n" +
+        "  IsDefault as isDefault, \n" +
+        "  AllowCombine as allowCombine, \n" +
+        "  IsFunctional as isFunctional, \n" +
+        "  AllowChange as allowChange,\n" +
+        "  IsSelectable as isSelectable, \n" +
+        "  IsVisible as isVisible, \n" +
+        "  studentControl as studentControl, \n" +
+        "  (select count(*) from configs.client_testtool TOOL where TOOL.ContextType = 'TEST' and TOOL.Context = MODE.testID and TOOL.clientname = MODE.clientname and TOOL.Type = TT.Type) as valCount, \n" +
```

Comment:
It is currently unused.  I wasn't sure when we wanted to remove the unused fields because I'm not 100% sure that the exam accommodations that is currently designed is has all the required information.